test/e2e/operator.go

```diff
@@ -463,3 +464,72 @@ var _ = Describe("ARO Operator - ImageConfig Reconciler", func() {
 		Eventually(verifyLists(nil, expectedBlocklist)).WithContext(ctx).Should(Succeed())
 	})
 })
+
+var _ = Describe("ARO Operator - dnsmasq", func() {
+	const (
+		timeout = 1 * time.Minute
+		polling = 10 * time.Second
+	)
+	mcpName := "test-aro-custom-mcp"
+	mcName := fmt.Sprintf("99-%s-aro-dns", mcpName)
+
+	customMcp := mcv1.MachineConfigPool{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "machineconfiguration.openshift.io/v1",
+			Kind:       "MachineConfigPool",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: mcpName,
+		},
+		Spec: mcv1.MachineConfigPoolSpec{},
+	}
+
+	getMachineConfigNames := func(g Gomega, ctx context.Context) []string {
+		machineConfigs, err := clients.MachineConfig.MachineconfigurationV1().MachineConfigs().List(ctx, metav1.ListOptions{})
+		g.Expect(err).NotTo(HaveOccurred())
+		names := []string{}
+		for _, mc := range machineConfigs.Items {
+			names = append(names, mc.Name)
+		}
+		return names
+	}
+
+	BeforeEach(func(ctx context.Context) {
+		By("Create custom MachineConfigPool")
+		_, err := clients.MachineConfig.MachineconfigurationV1().MachineConfigPools().Create(ctx, &customMcp, metav1.CreateOptions{})
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	AfterEach(func(ctx context.Context) {
```

Comment:
The e2e test cleans this up already to ensure the owner reference on the MCP is garbage collecting the MC that is created.  i don't think we actually need the `AfterEach` section with anything in it.  

The clusters that run e2e in our PRs and production pipelines are deleted after e2e completes.  This would only be applicable if someone was running multiple e2e tests against a single cluster they created.  Not sure if this is necessary.

Reply:
Removed the AfterEach definition.